pkg/serving/config_changes_test.go

```diff
@@ -139,6 +139,43 @@ func testUpdateEnvVarsModify(t *testing.T, revision *servingv1alpha1.RevisionTem
 	}
 }
 
+func TestUpdateContainerPort(t *testing.T) {
+	template, _ := getV1alpha1Config()
+	err := UpdateContainerPort(template, 8080)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Verify update is successful or not
+	for p, _ := range template.Spec.Containers {
```

Comment:
assert that there are only 1 container and 1 port. Then just assert on `template.Spec.Containers[0].Ports[0]`

Reply:
Modified